sdk/ai/Azure.AI.Inference/samples/Sample8_ChatCompletionsWithTelemetry.md

```diff
@@ -0,0 +1,145 @@
+# Using telemetry with Azure.AI.Inference
+
+In this example we will demonstrate how [OpenTelemetry](https://learn.microsoft.com/dotnet/core/diagnostics/observability-with-otel) library can be leveraged to diagnose the issues in the Chat completion request.
+
+## Project creation and dependency installation.
```

Comment:
```suggestion
## Project creation and dependency installation

```

Leaving one comment here but applies to all headers.

Reply:
Fixed